doc/tasks/pest.md

```diff
@@ -0,0 +1,36 @@
+# Pest
+
+The Pest task will run your unit tests.
+
+***Composer***
+
+```
+composer require --dev pestphp/pest
+```
+
+***Config***
+
+The task lives under the `pest` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+grumphp:
+    tasks:
+        pest:
```

Comment:
Would it make sense to allow the same options as the phpunit task?

Reply:
Hi,

The list of options provides by Pest is the same à PHPUnit, so yes, doing same here is a good idea :wink:

Reply:
PHPUnit provide `testsuite` and `group`, so i added it :smiling_face_with_three_hearts: